### PR TITLE
Make resources guidance more explicit

### PR DESCRIPTION
**Pull request by jesbis**
RE: #558 

@jevansaks, @chrisglein do you think a post-install readme update like this would also help make adding resources clearer?

### CODE REVIEW

**Comment by jevansaks** [comment]
> @jevansaks @chrisglein do you think a post-install readme update like this would also help make adding resources clearer?

Yes if we had a concrete example of what it looked like for someone who already had resources.

Also is "top" correct? I think the lookup order might be first-to-last and so bottom might be "last".

**Comment by chrisglein** [comment]
> Yes if we had a concrete example of what it looked like for someone who already had resources.
> Also is "top" correct? I think the lookup order might be first-to-last and so bottom might be "last".

Agreed that an example would be better here. Something that has the basic empty app XAML but room for
```
<!-- Your additional resources here, before XamlControlsResources -->

File: build/NuSpecs/readme.txt [hunk 1]
```diff
@@ -1,7 +1,22 @@
...
+
+or if you have other resources then add XamlControlsResources at the top as a merged dictionary:
 
     <Application.Resources>
-        <XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls"/>
+        <ResourceDictionary>
+            <ResourceDictionary.MergedDictionaries>
+                <XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls" />
+            </ResourceDictionary.MergedDictionaries>
+
```
Comment (chrisglein) [inline build/NuSpecs/readme.txt]: Add here:
```
<!-- Add your resources and styles here, AFTER XamlControlResources -->
```

**Comment by jesbis** [comment]
> Also is "top" correct? I think the lookup order might be first-to-last and so bottom might be "last".

Local resources should be "after"/"below" to take precedence, right?

**Comment by chrisglein** [comment]
> Also is "top" correct? I think the lookup order might be first-to-last and so bottom might be "last".
> 
> Local resources should be "after"/"below" to take precedence, right?

That's what I'm seeing through experimentation. In this example the Green style will take precedence, then Red (if green is removed), then the XamlControlsResources.

```
    <Application.Resources>
        <ResourceDictionary>
            <ResourceDictionary.MergedDictionaries>
                <ResourceDictionary>
                    <Style TargetType="MenuFlyoutPresenter">
                        <Setter Property="Background" Value="Blue"/>
                    </Style>
                </ResourceDictionary>
                
                <XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls"/>
                
                <ResourceDictionary>
                    <Style TargetType="MenuFlyoutPresenter">
                        <Setter Property="Background" Value="Red"/>
                    </Style>
                </ResourceDictionary>
            </ResourceDictionary.MergedDictionaries>

            <Style TargetType="MenuFlyoutPresenter">
                <Setter Property="Background" Value="Green"/>
            </Style>
        </ResourceDictionary>
    </Application.Resources>
```

**Pull request merged**